src/content/decentralized-social-networks.md

```diff
@@ -0,0 +1,94 @@
+
+Users use the platform's native ERC-20 token $MIND to pay for items. Users can also earn $MIND tokens by publishing popular content, contributing to the ecosystem, and referring others to the platform.
+
+## Ethereum and legacy social media platforms {#Ethereum-and-legacy-social-media-networks}
+
+Decentralized social media platforms aren't the only ones trying to incorporate blockchain technology. Many traditional sites are also planning to integrate Ethereum into their infrastructure:
+
+### Reddit {#reddit}
+
+Reddit has [touted Community Points](https://cointelegraph.com/news/reddit-to-reportedly-tokenize-karma-points-and-onboard-500m-new-users), which are ERC-20 token that users can earn by posting quality content and contributing to the online community. You can redeem these tokens to gain exclusive privileges and perks. For this project, Reddit is working with Arbitrum, a Layer-2 rollup designed to scale Ethereum transactions.
+
+### Twitter {#twitter}
+
+In January 2021, Twitter Blue [rolled out support for NFTs](https://mashable.com/article/twitter-blue-nft-profile-picture), allowing users to connect their wallets and display NFTs as profile pictures. The social media company has also [announced plans](https://www.theverge.com/2021/8/16/22627435/twitter-bluesky-lead-jay-graber-decentralized-social-web) to create a decentralized social network in future.
+
+### Instagram {#instagram}
+
+In May 2022, [Instagram announced support for NFTs](https://about.instagram.com/blog/announcements/instagram-digital-collectibles) on Ethereum and Polygon. Users can post NFTs directly to Instagram by connecting their Ethereum wallet. 
+
+## Use Ethereum-based social networks {#Use-Ethereum-social-media}
+
+**Status** - _Status is a secure messaging app that uses an open-source, peer-to-peer protocol, and end-to-end encryption to protect your messages from third parties._ 
+- [Website](https://status.im/)
+
+**Mirror** - _Mirror is a decentralized, user-owned publishing platform built on Ethereum that allows users to crowdfund ideas, monetize content, and build high-value communities. 
+- [Website](https://mirror.xyz/)
+
+## Further reading {#Further-reading}
+
+- [Social Networks Are the Next Big Decentralization Opportunity](https://www.coindesk.com/tech/2021/01/22/social-networks-are-the-next-big-decentralization-opportunity/) by Ben Goertzel
+- [Web3 holds the promise of decentralized, community-powered social networks](https://venturebeat.com/2022/02/26/web3-holds-the-promise-of-decentralized-community-powered-social-networks/) by Sumit Ghosh
+- [An Overview of the Blockchain Social Media Landscape](https://www.gemini.com/cryptopedia/blockchain-social-media-decentralized-social-media) by Gemini Cryptopedia
+- [How Blockchain Can Solve Social Media Privacy](https://www.investopedia.com/news/ethereum-blockchain-social-media-privacy-problem-linkedin-indorse/) by Prableen Bajpai
```

Comment:
Similar to DID page, if we're going to use the `use-case` template, we could consider adding a bit more here in the same format as the other use case pages. Similar to my comment on that PR, don't think it needs to be a blocker.